README.md

```diff
@@ -14,11 +14,13 @@ Once you're in as a Collaborator, it's time to choose which article you'd like t
 
 ### Claim Your Article
 
-Found the perfect piece to translate? Awesome! Comment on the issue or article saying, "Assign this article to me." We'll then assign it to you, and you'll be all set to work your magic.
+Found the perfect piece to translate? Awesome! Comment `/translate` on the issue. This command will automatically assign the issue to you and also move the card to "in Translation" status.
```

Comment:
Found the perfect piece to translate? Awesome! Just comment `/translate` on the issue. This will automatically assign the issue to you and move the card to the "in Translation" status.